### ISSUE TITLE
No LEDs have been configured error

### ISSUE THREAD

**Issue by 3aceskelsen** [jstrausd/homebridge-simple-wled#28]
I'm getting the following error. I'm not super proficient, so I'm struggling to find where the issue is. does the name matter? I just used a name I wanted to call it.
<img width="459" alt="Screen Shot 2021-11-28 at 3 06 44 PM" src="https://user-images.githubusercontent.com/3276331/143787872-ae8edb2e-241d-4f98-a31c-0b081b91e0d2.png">
<img width="412" alt="Screen Shot 2021-11-28 at 3 07 20 PM" src="https://user-images.githubusercontent.com/3276331/143787887-c189bb56-a410-4f33-8dd9-72f8edd049be.png">

.

**Comment by mhuca** [comment]
I'm having the same issue.

{
    "platform": "WLED",
    "name": "WLED Ceiling",
    "host": "10.0.0.161",
    "effects": [
        "Scan",
        "Scanner",
        "Candy Cane",
        "Andriod",
        "Rainbow Runner",
        "Circus",
        "Merry Christmas",
        "Fireworks"
    ],
    "log": true
}

**Comment by 3aceskelsen** [comment]
@mhuca what are you using as your host number? I was using the IP address of the WLED controller. Is that correct?

**Comment by mhuca** [comment]
@3aceskelsen 

Yes ... I'm using the IP address of the WLED Controller....

I'm able to ping the devices from Homebridge... (Bring up terminal and can launch a ping)

I've made it as simple as I can... but nothing appears in Accessories.

**Comment by mhuca** [comment]
@3aceskelsen 

Just a thought... what firmware are you using on the WLED controller... I'm using WLED version 0.13.0-b5

**Comment by jstrausd** [comment]
Can you all post your complete config.json please

**Comment by mhuca** [comment]
@jstrausd 

homebridge-simple-wled v1.2.4
Docker oznu/homebridge:latest
WLED version 0.13.0-b5
HomeBridge Version
Node.js Version v16.13.1
I had to remove platforms for "MQTTThing" and "Camera-ffmpeg"
{
    "bridge": {
        "name": "Homebridge",
        "username": "CC:22:3D:E3:CE:40",
        "port": 52503,
        "pin": "031-45-154",
        "advertiser": "ciao"
    },
    "description": "",
    "accessories": [
    
    ],
    "platforms": [
        {
            "mqtt": {
                "base_topic": "zigbee2mqtt",
                "server": "mqtt://10.0.0.36:1883",
                "reject_unauthorized": false,
                "keepalive": 60,
                "version": 4,
                "disable_qos": false
            },
            "defaults": {
                "exclude": false
            },
            "devices": [
                {
                    "id": "FamilyRoomBackRemote"
                },
                {
                    "id": "GuestBedroomLampshade"
                },
                {
                    "id": "0x001788010202a106"
                },
                {
                    "id": "0x001788010357b1c7"
                },
                {
                    "id": "WilliamBedroomRemote"
                },
                {
                    "id": "0x00158d0001cbbd6b"
                },
                {
                    "id": "WilliamsBedroomLamp"
                },
                {
                    "id": "0x0017880103738215"
                },
                {
                    "id": "0x001788010357b1c1"
                },
                {
                    "id": "0x00178801039a8f86"
                },
                {
                    "id": "0x00178801027d88de"
                },
                {
                    "id": "0x00178801027d8921"
                },
                {
                    "id": "0x00158d0007900c1d"
                },
                {
                    "id": "0x00158d000708032e"
                }
            ],
            "platform": "zigbee2mqtt"
        },
        {
            "platform": "WLED",
            "name": "WLED Ceiling",
            "host": "10.0.0.161",
            "effects": [
                "Scan",
                "Scanner",
                "Candy Cane",
                "Andriod",
                "Rainbow Runner",
                "Circus",
                "Merry Christmas",
                "Fireworks"
            ],
            "log": true
        }
    ],
    "disabledPlugins": [
        "homebridge-gsh"
    ]
}

**Comment by 3aceskelsen** [comment]
@jstrausd I'm on 1.2.4 for wled and running the latest version of HOOBS for my homebridge. My first post has my entire config file for WLED

**Comment by jstrausd** [comment]
Please look at the example config file: 

https://github.com/jstrausd/homebridge-simple-wled#-adding-the-accessory-to-the-configjson

You have to declare a wleds JSON-Array with the WLED instances in it. Refer to the example!

**Comment by mhuca** [comment]
OK I got it ..

I've added the "wleds": [ 
...
...
]
I know have item in the accessories area... Thank you for your help.

**Comment by 3aceskelsen** [comment]
@jstrausd I'm wondering if my host is incorrect. everyone else is using a 10.0.0... address, but I'm using the 192.168 address. This is probably super basic, but what host am I supposed to use?

**Comment by jstrausd** [comment]
It depends on your Router/Modem, what IP-adresses you have to use. But you have to use the IP-Adress for your host, with which you also can access the WLED Controller with the Web-Browser. (Just the IP-Adress of the WLED-Controller)

**Comment by 3aceskelsen** [comment]
@jstrausd When I use 192.168.5.14 in my browser it navigates to the WLED controller. Here is my config file again. It looks to me that everything is correct. compared to the example file. 
Does the "name" matter? I just used something that made sense to me.
<img width="430" alt="Screen Shot 2021-12-03 at 9 07 22 AM" src="https://user-images.githubusercontent.com/3276331/144634401-506092bf-a77d-44d3-8c06-9914a20dd691.png">

**Comment by jstrausd** [comment]
Please remove the second, „platform“: „WLED“ below.

**Comment by 3aceskelsen** [comment]
Not sure why, but when I save it adds it back in

**Comment by jstrausd** [comment]
Is this „Simple Wled Bridge“ your whole config.json?

**Comment by 3aceskelsen** [comment]
I'm using hoobs, and this is the config file I see when I go into the wled plugin.

**Comment by jstrausd** [comment]
![02EDD0E4-9732-4C86-9142-351FDEC440F6](https://user-images.githubusercontent.com/36381849/144636728-8b610baa-bb81-4a9a-bbf8-af61c588086d.jpeg)
Please try to only write only the configuration into the file which you see in the white rectangle

**Comment by 3aceskelsen** [comment]
Okay, in this other screen it let me remove the extra "platform". It looks like in the logs it's working now. But I'm still not seeing anything in Homekit. I've added the hub to one of my rooms, but that room is empty.

**Comment by 3aceskelsen** [comment]
still no luck. I've removed the bridge from homekit and added it again by scanning the qr code. My logs in homebridge say there is 1 platform running but no device show anywhere.
Does the "name" matter?

**Comment by 3aceskelsen** [comment]
@jstrausd I've progressed a bit today according to my log it appears to be working. But no matter what I do it's still not showing in homekit. I've followed that other post you mentioned and have re-added the hub to homekit.
Here is a screenshot of my logs

<img width="1066" alt="Screen Shot 2021-12-03 at 4 17 28 PM" src="https://user-images.githubusercontent.com/3276331/144685049-7641a2d6-9f7a-4d62-a30d-16755e500231.png">
